src/backend/expungeservice/form_filling.py

```diff
@@ -61,6 +61,13 @@
     "yamhill": "District Attorney - 535 NE 5th St #117 - McMinnville, OR 97128",
 }
 
+T = TypeVar("T")
+
+
+def join_dates_or_strings(arr=List[T], connector="; ", date_format="%b %-d, %Y") -> str:
+    get_attr = lambda elem: elem.strftime(date_format) if isinstance(elem, DateWithFuture) else elem
```

Comment:
I prefer to use nested functions over lambda's for things that you are naming (i.e., get_attr). And also not use the name `get_attr` even though this is really local in scope as `get_attr` as it feels too close to `getattr` that I have to think for a second whether or not it is intentional or just a mistake.

Reply:
Yes, let me clean that up. I clobbered some built in functions too that I should rename.

Reply:
Should be good to go now.